nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -670,6 +664,19 @@ After going through this checklist, you're ready to go!
 		else
 			to_chat(user, "<span class='notice'>You require [src] to be in maintenance mode to load munitions!.</span>")
 			return
+	if(istype(A, /obj/structure/overmap/fighter/prebuilt/escapepod) && ispath(has_escape_pod) && (!escape_pod || escape_pod?.loc != src))

```

Comment:
has_escape_pod sounds like it should be a boolean, not a path?
what is the point of has_escape_pod if there is also escape_pod?